app/controllers/providers/other_assets_controller.rb

```diff
@@ -0,0 +1,34 @@
+module Providers
+  class OtherAssetsController < BaseController
+    def show
+      @form = Citizens::OtherAssetsForm.new(model: declaration)
+    end
+
+    def update
+      @form = Citizens::OtherAssetsForm.new(form_params)
+      if @form.save
+        render plain: 'Navigate to next question after any other assets'
+      else
+        render :show
+      end
+    end
+
+    private
+
+    def legal_aid_application
```

Comment:
You don't need this method if you add `include Providers::ApplicationDependable`

Reply:
Done